scripts/lib/sphinx/sphinxHtmlToMarkdown.ts

```diff
@@ -84,6 +84,12 @@ export async function sphinxHtmlToMarkdown(options: {
       const dest = `${imageDestination}/${filename}`;
 
       $img.attr("src", dest);
+
+      if (isReleaseNotes && releaseNotesTitle) {
+        // Release notes links should point to the current version
+        $img.attr("src", dest.replace(/[0-9].*\//, ""));
```

Comment:
Interesting. What was happening before this change?

Reply:
Before this change, if we used the historical argument, the release notes had all the links correct but the images. The images were pointing to the historical folder instead of the current version. For example, `public/images/api/qiskit/0.44/circuit-1.png` instead of `public/images/api/qiskit/circuit-1.png`